src/main.rs

```diff
@@ -86,9 +43,10 @@ async fn main() -> anyhow::Result<()> {
         .subcommand(
             App::new("push")
                 .about("Packages and uploads Hippo artifacts, notifying Hippo")
-                .args(&bindle_build_args)
+                .args(bindle_build_args(true, false))
```

Comment:
This is very opaque at the call site, and unsafe as it would be easy for a careless refactor to mix them up.  In C# I would use a Flags enum here but I'm not sure what the Rust equivalent is - whether to have independent enums or what.

Reply:
A classic bitmask is an option, with something like `REQUIRE_BINDLE_URL` and `REQUIRE_DESTINATION_DIRECTORY`. I will poke about at the Rust docs and see what other options/conventions there might be.

Reply:
Alright, updated with an `enum`-based solution. I am testing out an alternative way of doing Clap args as part of the solution to #49. So this may just be a temporary fix if the struct-based opts work out.